docs/manual-EN/3.build-develop-and-administration/3.configurations/5.storage-config.md

```diff
@@ -43,11 +43,12 @@ nebula> UPDATE CONFIGS storage:v=1;
 
 > **NOTE**: We recommend you using the actual IP in the `meta_server_addrs` parameter because sometimes `127.0.0.1` will not be parsed correctly.
 
-## Consistency Seedings for storage
+## Consistency setting for storage
 
 | Property        | Default Value           | Default Value                |
 | ----------- | --------------------------------------------------- | -------------------------------------------------------- |
 | `data_path` | data/storage (i.e. `/usr/local/nebula/data/storage/`) |  Root consistent data path , multi paths should be split by comma. For RocksDB engine, one path one instance. |
```

Comment:
The root directory for the local data persistence? If multiple directories exist, use commas to separate the directories?